google-cloud-storage/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://mojo.codehaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+    <difference>
+        <className>com/google/cloud/storage/Storage</className>
+        <method>com.google.cloud.storage.Channel watchAll(java.lang.String, com.google.cloud.storage.Channel, com.google.cloud.storage.Storage$WatchAllOption[])</method>
+        <differenceType>7012</differenceType>
```

Comment:
Just out of curiosity, Do we really need to add differences XML even if we are introducing a fresh operation?

I think we are not breaking any exiting operation then why is it require? Do you any idea?

Reply:
correct here we aren't breaking any exiting operation, we are adding differences xml file to pass the `mvn clirr` checks.